Fixes and tests for color()

z3dev opened the pull request:
I think the comments on the commits are sufficient but let me know if there are any questions.

z3dev committed:
Expanded cases for 3D color tests

z3dev committed:
Corrected determination of arguments to color()
Reloaded CSS color map to internal object called cssColors
Exported new function css2rgb() to faciliate color lookups via CSS color name

kaosat-dev commented on src/color.js:
```diff
@@ -1,179 +1,201 @@
+ * @return  Array           The RGB representation, or [0,0,0] default
+ */
+export function css2rgb(s) {
+  var r = 0;
```
Everything about this PR is good, but I don't get this ? this would not default to [0,0,0], and r,g,b are not used ? 
a simple solution in this case :
```javascript
const defaults = [0,0,0]
const lookedUpColor = cssColors[s.toLowerCase()]
return lookedUpColor ? lookedUpColor : defaults
```

z3dev replied:
As debated off line, we both agreed that returning an error (exception) is better than returning a default color.

To throw an exception or to let the program die (color will be undefined) is another debate as the library needs to be consistent. Let's open a new issue to debate with others.

For now, this PR is consistent with the current policy for this function, i.e. let the program die.

kaosat-dev commented:
@z3dev  could you please just fix the small type (see tests) so I can merge? thanks :)

z3dev committed:
Corrected length of returned string in rgb2html()

z3dev committed:
Added missing imports
Added test for css2rgb()
Added test for html2rgb()
Added test for rgb2html()
Added failing to test cases with know issues
Corrected typo
